Database/Entities/Creature.cs

```diff
@@ -1,24 +1,82 @@
 ﻿namespace Database.Entities;
 
+/// <summary>
+/// Represents an in-game creature and it's data.
+/// Creatures belong to a specific faction and are categorized by tiers, with Tier 1 being the weakest 
+/// and Tier 7 being the strongest. <para/>
+/// 
+/// The <c>Creature</c> class provides properties to store and update the unique identifier, faction id,
+/// tier, base cost, upgrade cost, growth and image data for it's base, upgraded and alternate forms. <para/>
+/// 
+/// The binary image data stored in <c>ImageBytesBase</c>, <c>ImageBytesUpg</c> and <c>ImageBytesUpgAlt</c> 
+/// can be converted to a visual representation using the <c>System.Drawing.Image</c> class. <br/>
+/// Note that <c>System.Drawing</c> might require additional dependencies on some platforms. <para/>
+/// 
+/// <b>Additionally</b>, it includes a <c>Faction</c> entity to which it belongs to.
+/// </summary>
 public partial class Creature
 {
+    /// <summary>
+    /// Gets or sets the unique identifier.
+    /// This is a 32-bit signed integer.
+    /// </summary>
     public int ID { get; set; }
 
+    /// <summary>
+    /// Gets or sets the identifier of the faction the creature belongs to.
+    /// This is a 32-bit signed integer.
+    /// </summary>
     public int FactionID { get; set; }
 
+    /// <summary>
+    /// Gets or sets the raw binary image data for the creature's base form. 
+    /// <i>For example, a Peasant.</i><para/>
+    /// This can be used to reconstruct the image using libraries such as <c>System.Drawing</c>.
+    /// </summary>
```

Comment:
Line 33 is completely out of place, since "a Peasant" is not a raw binary image data.
Line 34 is covered by earlier comment.